pkg/controller/agent/fleet.go

```diff
@@ -265,6 +285,29 @@ func maybeReconcileFleetEnrollment(params Params, result *reconciler.Results) En
 	return token
 }
 
+func getKibanaBasePath(ctx context.Context, client k8s.Client, kibanaNSN types.NamespacedName) (string, error) {
+	var kb v1.Kibana
+	err := client.Get(ctx, kibanaNSN, &kb)
+	if err != nil {
+		return "", err
+	}
+
+	if kb.Spec.Config != nil {
```

Comment:
Can we flip this and just return early if `== nil`?  Then the remainder of the function is cleaner, and not in the middle of an if block.

Reply:
done.